### PR TITLE
report submission endpoint implemented

### PR DESCRIPTION
**Pull request by cupofocha**

### CODE REVIEW

File: app/db/user_db.py
Comment (csaba92b) [inline app/db/user_db.py:1]: These are implemented in fastapi_users manager class for e.g:
https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/manager.py#L79

File: app/models/__init__.py
Comment (csaba92b) [inline app/models/__init__.py:1]: whats that, and why in the __init__ file? :D

File: app/models/report_storage.py
Comment (csaba92b) [inline app/models/report_storage.py:1]: use beanie Document as parent class.
https://github.com/roman-right/beanie/
https://www.youtube.com/watch?v=0vBluFlNipQ&list=PLc0zQ1NGdHHLU3SDF-rjKEfBVgIiAusU-&index=3&t=2756s

File: app/settings.py [hunk 1]
```diff
@@ -3,13 +3,12 @@
 
 class Settings(BaseSettings):
     db_username: str
-    db_password: str 
-    db_cluster: str 
-    secret_key: str
+    db_password: str
+    db_cluster: str
+    secret_key: str = "123"
```
Comment (csaba92b) [inline app/settings.py]: bad practice, github will go crazy about this.

File: app/settings.py [hunk 2]
```diff
@@ -3,13 +3,12 @@
...
 
     class Config:
-        env_file = ".env"
-        
+        env_file = "./.env"
```
Comment (csaba92b) [inline app/settings.py]: its not a path, the parent class makes use of that, at least does not work like that on my side, (neither ./  .\ )

File: app/db/db.py [hunk 3]
```diff
@@ -5,21 +5,20 @@
...
+db_tokens = db["AccessToken"]
+db_reports = db["Report"]
 
 async def get_user_db():
     yield BeanieUserDatabase(User)
 
 
-
-async def get_access_token_db():  
+async def get_access_token_db():
```
Comment (csaba92b) [inline app/db/db.py:21]: make one for the reports as well :D

File: app/db/report_storage_db.py
Comment (csaba92b) [inline app/db/report_storage_db.py:1]: copilot was in charge today :D

**Review by csaba92b — COMMENTED** [review]
I had to submit the review to let you see the comments.

File: app/db/user_db.py
Comment (cupofocha) [reply app/db/user_db.py:1]: > These are implemented in fastapi_users manager class for e.g: https://github.com/fastapi-users/fastapi-users/blob/master/fastapi_users/manager.py#L79
@csaba92b nice, i was looking for this, couldn't found it.

File: app/models/__init__.py
Comment (cupofocha) [reply app/models/__init__.py:1]: forgot to delete it :P

File: app/settings.py (hunk 1 above)
Comment (cupofocha) [reply app/settings.py]: yeah, didn't want to commit this file, i did somehow

File: app/settings.py (hunk 2 above)
Comment (cupofocha) [reply app/settings.py]: the original path didn't work for me, no matter where i put the .env file, anyway, i will change it back

File: app/db/report_storage_db.py
Comment (cupofocha) [reply app/db/report_storage_db.py:1]: it got the works done, no?

File: app/models/report_storage.py
Comment (cupofocha) [reply app/models/report_storage.py:1]: yeah... i will save it for tmr, one more doc is going to kill me

File: app/db/db.py (hunk 3 above)
Comment (cupofocha) [reply app/db/db.py:21]: @csaba92b they are built-in functions from fastapi-users, no? I got a more simplified way getting the db on above
`db_reports = db["Report"]`
Edit: nvm just realized how the whole thing works

**Review by csaba92b — APPROVED** [review]
alles gut!

**Pull request merged**